crates/optimism/src/l1block.rs

```diff
@@ -17,10 +17,10 @@ const BASE_FEE_SCALAR_OFFSET: usize = 16;
 /// The two 4-byte Ecotone fee scalar values are packed into the same storage slot as the 8-byte sequence number.
 /// Byte offset within the storage slot of the 4-byte blobBaseFeeScalar attribute.
 const BLOB_BASE_FEE_SCALAR_OFFSET: usize = 20;
-/// The two 8-byte Holocene operator fee scalar values are similarly packed. Byte offset within
+/// The two 8-byte Isthmus operator fee scalar values are similarly packed. Byte offset within
```

Comment:
oops I forgot this during our call earlier.

In this version of the PR, I packed the operator fee parameters into the same slot as the ecotone scalars. Since then, the spec has been revised to move the operator fee params into their own slot. I think the optimism monorepo PR should have this logic somewhere?

For whatever reason you don't have to specify the slots of different parameters in the spec, so I didn't include it there ... Just choose a slot for these operator fee parameters.

Also this PR doesn't include the operatorFeeVault or the operatorFeeManager so make sure that those are included where necessary.